examples/nidcpower_source_dc_voltage_with_multiplexer/measurement.py

```diff
@@ -12,7 +12,7 @@
 import click
 import grpc
 import hightime
-import ni_measurementlink_service as nims
+import ni_measurement_plugin as nims
```

Comment:
Consider removing the "nims" alias in a separate PR

Reply:
Will do

Reply:
Cristina informed me the new name is `ni_measurement_plugin_sdk`. With this change 'nims' could be interpreted as 'NI Measurement SDK'. Do we still want to change this to, e.g. `as nimp` or something similar or just leave it?

Reply:
Given the updated name, I think leaving it as `nims` is fine.

FYI: I'd prefer a name that doesn't need to be aliased or abbreviated, but I guess that's not going to happen. At least `ni_measurement_plugin_sdk` is 1 letter shorter than `ni_measurementlink_service`.

Are we getting any benefit out the alias? It looks like we are:
```
examples\nidcpower_source_dc_voltage\measurement.py
30:measurement_service = nims.MeasurementService(
52:    nims.DataType.IOResourceArray1D,
54:    instrument_type=nims.session_management.INSTRUMENT_TYPE_NI_DCPOWER,
56:@measurement_service.configuration("voltage_level", nims.DataType.Double, 6.0)
57:@measurement_service.configuration("voltage_level_range", nims.DataType.Double, 6.0)
58:@measurement_service.configuration("current_limit", nims.DataType.Double, 0.01)
59:@measurement_service.configuration("current_limit_range", nims.DataType.Double, 0.01)
60:@measurement_service.configuration("source_delay", nims.DataType.Double, 0.0)
61:@measurement_service.output("measurement_sites", nims.DataType.Int32Array1D)
62:@measurement_service.output("measurement_pin_names", nims.DataType.StringArray1D)
63:@measurement_service.output("voltage_measurements", nims.DataType.DoubleArray1D)
64:@measurement_service.output("current_measurements", nims.DataType.DoubleArray1D)
65:@measurement_service.output("in_compliance", nims.DataType.BooleanArray1D)
```

Reply:
Expanding these would push most of them over the 100-column mark, requiring more wrapping.

```
examples\nidcpower_source_dc_voltage\measurement.py
30:measurement_service = ni_measurement_plugin_sdk.MeasurementService(
52:    ni_measurement_plugin_sdk.DataType.IOResourceArray1D,
54:    instrument_type=ni_measurement_plugin_sdk.session_management.INSTRUMENT_TYPE_NI_DCPOWER,
56:@measurement_service.configuration("voltage_level", ni_measurement_plugin_sdk.DataType.Double, 6.0)
57:@measurement_service.configuration("voltage_level_range", ni_measurement_plugin_sdk.DataType.Double, 6.0)
58:@measurement_service.configuration("current_limit", ni_measurement_plugin_sdk.DataType.Double, 0.01)
59:@measurement_service.configuration("current_limit_range", ni_measurement_plugin_sdk.DataType.Double, 0.01)
60:@measurement_service.configuration("source_delay", ni_measurement_plugin_sdk.DataType.Double, 0.0)
61:@measurement_service.output("measurement_sites", ni_measurement_plugin_sdk.DataType.Int32Array1D)
62:@measurement_service.output("measurement_pin_names", ni_measurement_plugin_sdk.DataType.StringArray1D)
63:@measurement_service.output("voltage_measurements", ni_measurement_plugin_sdk.DataType.DoubleArray1D)
64:@measurement_service.output("current_measurements", ni_measurement_plugin_sdk.DataType.DoubleArray1D)
65:@measurement_service.output("in_compliance", ni_measurement_plugin_sdk.DataType.BooleanArray1D)
```